### PR TITLE
Update dependencies and minor cleanup

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,5 +86,5 @@ For more information on semantic versioning, please visit http://semver.org/.
 License
 -------
 
-Copyright (c) 2013 [Jake Harding](http://thejakeharding.com)
```
Comment (jharding) [inline README.md:87]: Can you revert this change as it's needed to insert a newline.

File: tasks/sed.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 module.exports = function(grunt) {
   var replace = require('replace')
-    , _ = grunt.util._
+    , _ = require('lodash')
```
Comment (jharding) [inline tasks/sed.js]: What's gained from relying on lodash directly?

Comment (XhmikosR) [inline tasks/sed.js]: http://gruntjs.com/api/grunt.util#grunt.util._

File: README.md (hunk 1 above)
Comment (XhmikosR) [inline README.md:87]: I went with a clearer approach using `<br/>`

**Comment by XhmikosR** [comment]
I rebased the branch to keep only the package.json changes; I'll make a new PR for the rest after this is merged.
